pkg/subctl/cmd/deploybroker.go

```diff
@@ -20,15 +20,20 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	lighthouse "github.com/submariner-io/submariner-operator/pkg/subctl/lighthouse/deploy"
```

Comment:
This should be grouped with the Submariner imports in the following section.

Reply:
I expected the plugin in IDE to take care of this, looks like it doesn't. Done.